### PR TITLE
add a management-command to bootstrap an App

### PR DESCRIPTION
**Pull request by sravfeyn**
Whenever you want an App (e.g. doing some UI changes on app-views) this command can create it. I found myself creating test apps too many times when developing app-related features (at least for UI related changes).

### CODE REVIEW

File: corehq/apps/domain/management/commands/bootstrap_app.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+class Command(LabelCommand):
+    help = "Bootstrap an Application with two modules in a given domain."
+    args = "<domain> <app-name>"
+    label = ""
+
+    def handle(self, *args, **options):
+
+        if len(args) != 2:
+            raise CommandError('Usage: manage.py bootstrap_app <domain> <app-name>')
+        domain_name, app_name = args
```
Comment (sravfeyn) [inline corehq/apps/domain/management/commands/bootstrap_app.py]: Ideally, one would wish to pass an App-config and let this command create the app. But, that will be too much effort for now, this is a good starting point, I think.

**Comment by sravfeyn** [comment]
@esoergel or anyone

File: corehq/apps/domain/management/commands/bootstrap_app.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/domain/management/commands/bootstrap_app.py]: Yeah, it seems out of scope if you're using this for test apps where that doesn't matter

**Comment by benrudolph** [comment]
this is sweet. it'd be cool to also be able to specify different properties in the config, but that could be getting a little complex
